arrays/arrays.c

```diff
@@ -84,15 +111,25 @@ char *arr_read(Array *arr, int index) {
 void arr_insert(Array *arr, char *element, int index) {
 
   // Throw an error if the index is greater than the current count
-
+  if (index > arr->count)
+  {
+    printf("Index is greater than current count.");
+    exit(1);
+  }
   // Resize the array if the number of elements is over capacity
-
+  if (arr->count + 1 > arr->capacity)
```

Comment:
jaja nice thinking! could also do `arr->count >= arr->capacity`, or even `arr->count == arr->capacity` if you're brave enough